holoviews/plotting/bokeh/element.py

```diff
@@ -533,9 +573,9 @@ def initialize_plot(self, ranges=None, plot=None, plots=None, source=None):
             self._update_glyph(glyph, properties, mapping)
         if not self.overlaid:
             self._update_plot(key, plot, style_element)
-        if self.callbacks:
-            self.callbacks(self)
-            self.callbacks.update(self)
+
+        for cb in self.callbacks:
+            cb.initialize()
         if not self.overlaid:
```

Comment:
It is a little confusing that there is a ``_init_callbacks`` method which actually constructs the callback instances (based on my understanding of it) and then an ``initialize`` method as well. Maybe the former should be called ``_construct_callbacks`` instead...